src/components/send/SendButton.js

```diff
@@ -39,7 +39,7 @@ export default function SendButton({
       isAuthorizing={isAuthorizing}
       label={
         disabled && requiresChecks
-          ? `􀄨 ${lang.t('wallet.transaction.complete_checks')}`
+          ? `􀄨 ${lang.t('wallet.transaction.complete_check')}`
```

Comment:
this could be plural with ENS p sure, we may need to count the checks

Reply:
ahh yeah i'll look at it

Reply:
@skylarbarrera 
All the ENS checks are optional so it actually doesn't apply here. 
<img width="352" alt="Screenshot 2024-04-17 at 10 29 30 AM" src="https://github.com/rainbow-me/rainbow/assets/41711440/3ec2492c-1ede-434c-9d86-052579179629">

Reply:
also i forgot about this PR lol